include/ocpp/v201/charge_point.hpp

```diff
@@ -166,6 +161,22 @@ struct Callbacks {
     /// \brief Callback function that can be used to handle arbitrary data transfers for all vendorId and
     /// messageId
     std::optional<std::function<DataTransferResponse(const DataTransferRequest& request)>> data_transfer_callback;
+
+    /* Callbacks for networking */
+    /// \brief register a \p callback that is called when the websocket is connected successfully
+    std::optional<std::function<void(const std::optional<NetworkConnectionProfile>& network_connection_profile)>>
```

Comment:
In v16 there is this callback:
    `void register_connection_state_changed_callback(const std::function<void(bool is_connected)>& callback);`
It would be nice to have it synced with 1.6, however, I like the NetworkConnectionProfile here because it is nice to know to which profile we are connected now.